README.md

```diff
@@ -298,6 +298,47 @@ by its ID with the `custom.` prefix:
 />
 ```
 
+### Custom Queries within catalog-info.yaml file of the Backstage Entity
+
+You can also register your custom queries in the catalog-info.yaml file.
+
+```yaml
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: demo-backstage
+  description: Backstage Demo instance.
+  annotations:
+    backstage.io/kubernetes-id: kubernetescustom
+spec:
+  type: website
+  owner: user:default/mjakl
+  lifecycle: experimental
+  system: integrations
+  queries:
+    - name: Fetch all Davis events 1
```

Comment:
Can we exchange this example with another one, for instance, the PROBLEM_EVENTS from below?

The reason is that I want to avoid confusion on what DAVIS Events are.